test/Microsoft.ML.Tests/Scenarios/Api/CookbookSamples/CookbookSamples.cs

```diff
@@ -562,6 +562,9 @@ private void CrossValidationOn(string dataPath)
                     Label: r.Label.ToKey(),
                     // Concatenate all the features together into one column 'Features'.
                     Features: r.SepalLength.ConcatWith(r.SepalWidth, r.PetalLength, r.PetalWidth)))
+                // Add a step for caching data in memory so that the downstream iterative training
+                // algorithm can efficiently scan through the data multiple times.
+                .AppendCacheCheckpoint()
```

Comment:
>AppendCacheCheckpoint [](start = 17, length = 21)

you need to update [https://github.com/dotnet/machinelearning/blob/master/docs/code/MlNetCookBook.md](https://github.com/dotnet/machinelearning/blob/master/docs/code/MlNetCookBook.md) with these changes. (here and in SamplesDynamicApi)

Reply:
Done. Could you take a look again?

---
In reply to: [238883605](https://github.com/dotnet/machinelearning/pull/1780#discussion_r238883605) [](ancestors = 238883605)